weave/artifact_wandb.py

```diff
@@ -457,6 +457,30 @@ def __repr__(self):
     def delete(self) -> None:
         self._saved_artifact.delete(delete_aliases=True)
 
+    def add_artifact_reference(self, uri: str) -> None:
+        # The URI for a get node might look like: wandb-artifact:///<entity>/<project>/test8:latest/obj
+        # but the SDK add_reference call requires something like:
```

Comment:
nit: this comment seems to imply a conversion to numeric id... but the code does something different

Reply:
It's numeric after we get the `ref_url()` below.

Reply:
ahhh - makes sense - ty